operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/polling/PollingEventSource.java

```diff
@@ -18,17 +18,33 @@
   private final Timer timer = new Timer();
   private final Supplier<Map<ResourceID, T>> supplierToPoll;
   private final long period;
+  private final long initialDelay;
 
+  /**
+   * The initial delay can be configured, however the default is set to period since on startup
+   * operator will reconcile so the first should happen naturally on period.
```

Comment:
This sentence isn't clear.

Reply:
just uipdated and added javadoc to the class